backend/README.md

```diff
@@ -87,10 +98,14 @@ make testmigratedown
 
 #### Dealing with Dirty Schema Migrations
 
+Reset Schema Migrations
+```bash
+UPDATE schema_migrations SET dirty = false
+```
 
 ### Testing
 
-1. Run `flow emulator` from this top-level directory (the private key for the `emulator-account` in `flow.json` must match the private key hard-coded in the test suite). Install the `flow` CLI [here](https://docs.onflow.org/flow-cli/install/)
+1. Run `flow emulator --dev-wallet` from this top-level directory (the private key for the `emulator-account` in top level `flow.json` must match the private key hard-coded in the test suite eg. `backend/cadence/V2/tests/flow.json`). Install the `flow` CLI [here](https://docs.onflow.org/flow-cli/install/)
```

Comment:
I don't think this is required for backend tests. @rvmelkonian @jacksonConrad can you comment on this

Reply:
yes that's correct actually its only required as a flag when running the front end, so maybe best to leave it as an optional flag for now